src/prefect/tasks/kubernetes/job.py

```diff
@@ -613,9 +614,9 @@ class RunNamespacedJob(Task):
         - kubernetes_api_key_secret (str, optional): the name of the Prefect Secret
             which stored your Kubernetes API Key; this Secret must be a string and in
             BearerToken format
-        - job_status_pool_interval (int, optional): The interval given in seconds
-            indicating how often the Kubernetes API will be requested about the status
-            of the job being performed, defaults to the `5` seconds
+        - log_level (str, optional): Log level used when outputting logs from the job
```

Comment:
Some jobs will generate lots of logs, and not all users will want logs stored in prefect's backend. This should default to `None` IMO (so it's opt-in).